demos/spatio-temporal/input_data.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""
+Updated on  Wed Mar 11 09:28:50 2020
+
+@author: 
+"""
+
+import numpy as np
+import pandas as pd
+import pickle as pkl
+
+def load_sz_data(dataset):
+    sz_adj = pd.read_csv(r'data/sz_adj.csv',header=None)
```

Comment:
What's the `sz` data?

Reply:
This is the alternative dataset of trajectories of taxis. I had it as alternative but not using it. I will remove it.